lite/tests/detection/test_detailed_counts.py

```diff
@@ -969,6 +789,30 @@ def test_detailed_counts_using_torch_metrics_example(
 
     assert len(metrics) == 6
 
+    uid0_gt_0 = ("0", 214.125, 562.5, 41.28125, 285.0)
```

Comment:
can you update the output format so that the coordinates are wrapped in a list or tuple? this will help us provide consistent outputs in a world where we move beyond bounding boxes

Reply:
The examples are always going to be boxes so im not sure what you mean with this?

In saying uid0_gt_0 this is referencing the 0-index groundtruth from datum with uid0 in this example.

We cant directly reference the fixture data because i am storing the example boxes in a reduced floating-point precision to save on memory use.

Reply:
TODO add a card to the board to remind yourself to document the floating point precision change for n_samples